package.json

```diff
@@ -6,7 +6,7 @@
   "author": "Vitor Balocco",
   "repository": "eslint/eslint-transforms",
   "engines": {
-    "node": ">=4"
+    "node": "^12.22.0 || ^14.17.0 || >=16.0.0"
```

Comment:
This is a breaking change for this package.

Reply:
Which, I think, is fine but we should update the commit tag to `feat!` to bump major version on release.